dvc/command/remote.py

```diff
@@ -108,6 +108,34 @@ def run(self):
         return 0
 
 
+class CmdRemoteRename(CmdRemote):
+    def run(self):
+        conf = self.config.load_one(self.args.level)
+        self._check_exists(conf)
```

Comment:
Nitpicky/minor issue, but we end up loading this conf again when you do
```py
with self.config.edit(self.args.level) as conf:
```

So if we move the `_check_exists` call and the code for checking if new name exists in all/other config levels into the `with` statement, we can get rid of the extra `load_one` call. Something like:

```py
with self.config.edit(self.args.level) as conf:
    self._check_exists(conf)

    # <check if new name exists in all/other levels here>

    conf["remote"][self.args.new] = conf["remote"][self.args.name]
    del conf["remote"][self.args.name]